cancer_ai/validator/competition_manager.py

```diff
@@ -66,6 +77,37 @@ def __init__(
         self.hotkeys = []
         self.chain_miner_models = {}
 
+    def log_results_to_wandb(
+        self, hotkey: str, evaluation_result: ModelEvaluationResult
+    ) -> None:
+        wandb.init(project=self.config.wandb_project_name)
+        wandb.log(
+            {
+                "hotkey": hotkey,
+                "tested_entries": evaluation_result.tested_entries,
+                # "model_test_run_time": evaluation_result.run_time,
+                "accuracy": evaluation_result.accuracy,
+                "precision": evaluation_result.precision,
+                "recall": evaluation_result.recall,
+                "confusion_matrix": evaluation_result.confusion_matrix.tolist(),
+                "roc_curve": {
+                    "fpr": evaluation_result.fpr.tolist(),
+                    "tpr": evaluation_result.tpr.tolist(),
+                },
+                "roc_auc": evaluation_result.roc_auc,
+            }
+        )
+
+        wandb.finish()
+        print("Logged results to wandb")
```

Comment:
prints instead of logger? Can we aggregate all of these logs into one log statement if possible?